doc/src/cylc-user-guide/cug.tex

```diff
@@ -5324,6 +5324,59 @@ \subsubsection{Raising Exceptions}
 {{ assert(VARIABLE is defined, 'VARIABLE must be defined for this suite.') }}
 \end{lstlisting}
 
+\subsection{EmPy}
+\label{EmPy}
+
+In addition to Jinja2, Cylc supports EmPy template processor in suite
+configurations. Similarly to Jinja2, EmPy provides variables, mathematical
+expressions, loop control structures, conditional logic, etc., that are expanded
+to generate the final suite configuration seen by cylc. See
+\href{http://www.alcyone.com/software/empy}{EmPy documentation} for more details
+on its templating features and how to use them. Please note that EmPy is not
+bundled with Cylc and must be installed separately. It should be available to
+Python through standard \lstinline[language=Python]=import em=. Please also
+note that there is another Python package called ``em'' that provides
+a conflicting module of the same name. You can run
+\lstinline=cylc check-software= command to check your installation.
+
+The need for Empy processing must be declared with a hash-bang comment as
+the first line of the suite.rc file:
+\begin{lstlisting}
+#!empy
+# ...
+\end{lstlisting}
+
+An example suite \lstinline=empy.cities= demostrating its use is shown below.
+It is a translation of \lstinline=jinja2.cities= example from
+Section~\ref{Jinja2} and can be directly compared against it.
+
+\lstset{language=suiterc}
```

Comment:
eh?

Reply:
I can't reply to your review summary, but let me just register a downvote at the use of the phrase '*lesser* tools'.

Reply:
@oliver-sanders - what's your "eh?" refer to? The phrasing of "can be directly compared against it."? (should be "compared with it" I guess).

Reply:
Sorry, it was the `lstset` command.

Reply:
What's wrong with the `lstset` command?

Reply:
Nothing, I was reading it wrong.